p2p/host.go

```diff
@@ -52,16 +48,14 @@ type Peer struct {
 	PeerID          libp2p_peer.ID // PeerID, the pubkey for communication
 }
 
-func (p Peer) String() string {
-	BLSPubKey := "nil"
-	if p.ConsensusPubKey != nil {
-		BLSPubKey = p.ConsensusPubKey.SerializeToHexStr()
-	}
-	return fmt.Sprintf(
-		"BLSPubKey:%s-%s/%s[%d]", BLSPubKey,
-		net.JoinHostPort(p.IP, p.Port), p.PeerID, len(p.Addrs),
-	)
-}
+const (
+	// SetAsideForConsensus ..
+	SetAsideForConsensus = 1 << 14
```

Comment:
These concurrency number seems very high.

Reply:
reduced to 1<<13